upload/checks.py

```diff
@@ -1,9 +1,27 @@
 from functools import lru_cache, partial
 from goodtables import check
-from upload.constants import skip_list, synonym_to_chebi_name_dict, compound_skip
 from potion_client.exceptions import ItemNotFound
 import gnomic
 
+from upload.constants import skip_list, synonym_to_chebi_name_dict, compound_skip
+from upload import iloop_client, logger
+from upload.settings import Default
+
+
+IDENTIFIER_TYPES = {'protein', 'reaction'}
+
+
+def load_identifiers(type):
+    iloop = iloop_client(Default.ILOOP_API, Default.ILOOP_TOKEN)
+    return set(iloop.BiologicalIdentifier.subset(type=type))
```

Comment:
Could make this and the `IDENTIFIER_TYPES` plus `UPLOAD_TYPES` `frozenset`s so they can't be modified but it's certainly not necessary.